Doc/library/stdtypes.rst

```diff
@@ -2020,6 +2020,9 @@ expression support in the :mod:`re` module).
    *new*.  If the optional argument *count* is given, only the first *count*
    occurrences are replaced.
 
+   .. versionchanged:: 3.13
+      *count* is now supported as a keyword argument.
```

Comment:
Maybe change the signature to ``.. method:: str.replace(old, new, count=-1)`` and document that -1 means replace all occurrences, as documented in the docstring.

Reply:
Good idea, updated!